### PR TITLE
Merged latest updates from Code-11-SP3

### PR DESCRIPTION
**Pull request by mchf**

### CODE REVIEW

File: yast2-network.spec.in [hunk 1]
```diff
@@ -8,9 +8,8 @@ BuildRequires:  yast2-devtools yast2-testsuite
 BuildRequires:  yast2-country
 BuildRequires:  yast2-installation >= 2.15.27
 
-# NetworkInterfaces::Write should write ifcfgs only once
-BuildRequires:	yast2 >= 2.17.120
-Requires:	yast2 >= 2.17.120
+BuildRequires:	yast2 >= 2.17.135.1
+Requires:	yast2 >= 2.17.135.1
```
Comment (mvidner) [inline yast2-network.spec.in]: 2.17.137

**Comment by mvidner** [comment]
Please fix the dependency version, then it LGTM.

**Comment by jreidinger** [comment]
looks good for me

**Comment by mchf** [comment]
Dependency fixed, merging. Thanks for reviews.

**Pull request merged**